.gitmodules

```diff
@@ -2,7 +2,7 @@
   path = spack
   #url = https://github.com/spack/spack
   #branch = develop
-  url = https://github.com/jcsda/spack
+  url = https://github.com/JCSDA/spack
```

Comment:
This change isn't necessary, can you revert please (after successful CI runs, no need to rerun CI)?

Reply:
will do